### ISSUE TITLE
IDT2 includes on amp - Unescaped characters in alt text

### ISSUE THREAD

**Issue by greenc05** [bbc/simorgh#6998]
**Describe the bug**
There are unescaped characters in alt text on IDT2 includes on amp, this reads out character by character on a screen reader

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to https://www.bbc.com/gahuza/48118437.amp
2. Inspect the alt text for the 1st include
3. You will see unescaped characters in the alt text 
alt=“Afrika y```&#39;```epfo irimwo ubusumbane kurusha ahandi. Gini score.  Ibiharuro vyo mu 2014 kiretse  Namibia (2015) na Amerika (2013).”
4. **&#39;** reads out as  'and number 39' on a screen reader

**Expected behaviour**
Character codes should be ecsaped in alt text

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**
